cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -861,7 +861,7 @@ def _add_head_node(self):
         for device_index in range(1, head_node.max_network_interface_count):
```

Comment:
minor - should be this `network_interface_index` as it is for the compute nodes?

Reply:
Done